packages/mcp/package.json

```diff
@@ -22,6 +22,7 @@
     "lint-fix": "yarn sf-lint --fix",
     "package": "yarn pack",
     "postpack": "sf-clean --ignore-signing-artifacts",
+    "prepack": "sf-prepack",
```

Comment:
TODO: this scripts keeps getting added by dev-scripts, should be removed bc it breaks publish workflows